tools/cli/commands.js

```diff
@@ -1351,7 +1351,9 @@ main.registerCommand({
     // people to deploy from checkout or do other weird shit. We are not
     // responsible for the consequences.
     'override-architecture-with-local' : { type: Boolean },
-    'allow-incompatible-update': { type: Boolean }
+    'allow-incompatible-update': { type: Boolean },
+    'deploy-polling-timeout': {type: Number},
```

Comment:
Spacing on this line is inconsistent with the other lines in the PR itself. I honestly don't know what meteor/meteor style is (and it's a shame to have to work on project's without automatically enforced style guides) but internal inconsistency within a change is always good.